### DISCUSSION TITLE
Discussion on detecting scene from multiple shots

### DISCUSSION THREAD

**Discussion by AnaRhisT94** [Breakthrough/PySceneDetect#391] [General]
Hi,

I'm trying to think of an algorithm which can use PySceneDetect's shot detection and combine these shots into one single scene.
For example if I have 3 different scenes in a video and 9 shots that I detected with your package, lets assume every 3 consecutive shots are from the same scene.
Is there a way to associate these 3 shots into one single scene? Or how would you approach this?

**Comment by 4thanks** [comment]
use [ImageMagick](https://github.com/ImageMagick/ImageMagick)
```
cd "dir"
magick montage ./*.jpg -title 10x6 -geometry 180x320 test.jpg
```
